mongodbatlas/clusters.go

```diff
@@ -114,6 +114,60 @@ type clustersResponse struct {
 	TotalCount int       `json:"totalCount,omitempty"`
 }
 
+// DefaultDiskSizeGB represents the Tier and the default disk size for each one
+// it can be use like: DefaultDiskSizeGB["AWS"]["M10"]
+var DefaultDiskSizeGB map[string]map[string]float64 = map[string]map[string]float64{
```

Comment:
Could you remove it again please? it isn't already necessary in the Terraform part.

Reply:
Done

Reply:
Thank you so much 👍